menu-js-component/index.html

```diff
@@ -0,0 +1,185 @@
+<html>
+    <head>
+        <title>The Home Depot</title>
+        <link rel="stylesheet" href="styles.css">
+        <meta charset="UTF-8">
+    </head>
+    <body>
+        <header id="top-header">
+            <banner>
+                <p>Get Black Friday Prices Now Through December. While Supplies Last. Shop Now.</p>
+            </banner>
+
+            <!-- Main Menu - Top Menubar -->
+            <div class="main-menu-container">
+                <ul id="main-menu" class="main-menu">
+                    <li><a href="#" id="all-departments" class="drop-button">All Departments</a></li>
+                    <li><a href="#" id="home-decor-furniture" class="drop-button">Home Decor, Furniture & Kitchenware</a></li>
+                    <li><a href="#" id="diy" class="drop-button">DIY Projects & Ideas</a></li>
+                    <li><a href="#" id="project-calculators" class="drop-button">Project Calculators</a></li>
+                    <li><a href="#" id="installation-services" class="drop-button">Installation & Services</a></li>
+                    <li><a href="#" id="specials-offers" class="drop-button">Specials & Offers</a></li>
+                    <li><a href="#" id="local-ad-catalog" class="drop-button">Local Ad & Catalog</a></li>
+                </ul>
+            </div>
+
+            <!-- Level-1 Menus - Panel 1 - Dropdown -->
+
+            <div id="submenu-all-departments" class="submenu submenu-level-1 submenu-all-departments">
+                <ul>
+                    <li><a id="holiday-decorations" class="slideout-button" href="#">Holiday Decorations - subm</a></li>
+                    <li><a id="appliances" class="slideout-button" href="#">Appliances - has submenu</a></li>
```

Comment:
is the slideout-button class necessary on these elements, or can you grab them via the parent element ?